lib/node/nodes/dot-density.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var dot = require('dot');
+dot.templateSettings.strip = false;
+
+var Node = require('../node');
+
+var TYPE = 'dot_density';
+var PARAMS = {
+    source: Node.PARAM.NODE(Node.GEOMETRY.POLYGON),
+    points_col_name: Node.PARAM.NULLABLE(Node.PARAM.STRING),
```

Comment:
@stuartlynn have in mind that in some cases the editor will take care of that and will display just one analysis although it might be using two behind the scenes. Think about _area of influence_ where we use `trade-area` or `buffer` analyses depending whether is time or distance.

IMO we should split this analysis into two as I proposed.